tests/util/helpers.py

```diff
@@ -180,6 +182,30 @@ def deleteWorkspaceNoThrow(wsName: str):
         pass
 
 
+def workspacesEqual(Workspace1: str, Workspace2: str, **other_options):
+    if other_options == {}:
+        assert_wksp_almost_equal(Workspace1, Workspace2)
```

Comment:
This should return `True` or the other branch shouldn't return anything. At the moment, this code will run both functions if the first one doesn't throw an `AssertionError`.